### PR TITLE
Essential common components

### PR DESCRIPTION
**Pull request by tsiklic1**
Adding common components: Button, Input, Layout, Modal, Table

### CODE REVIEW

**Review by bdeak4 — CHANGES_REQUESTED** [review]

File: apps/admin/src/components/Modal/Modal.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import Button from "../Button";
+import c from "./Modal.module.scss";
+
+type ModalProps = {
+  children: React.ReactNode;
+  modal: boolean;
```
[the remark this answers is not in the document]
Comment (tsiklic1) [reply apps/admin/src/components/Modal/Modal.tsx:6]: ✅

**Comment by dujesa** [comment]
Formatirat css

File: apps/admin/src/App.tsx [hunk 2]
```diff
@@ -1,35 +1,18 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import "./App.scss";
+import Layout from "./components/Layout";
+import { Route } from "wouter";
+import HompePage from "./pages/HomePage";
+import GuestPage from "./pages/GuestPage";
+import EventsPage from "./pages/EventsPage";
```
Comment (AntimonDalmatinac) [inline apps/admin/src/App.tsx:6]: **
1_OARpkeBkn_Tw3vk8H769OQ
ovo mos koristit**

**Review by AntimonDalmatinac — CHANGES_REQUESTED** [review]
triba vise od 1% docker tehnologije

File: apps/admin/src/App.tsx (hunk 2 above)
Comment (tsiklic1) [reply apps/admin/src/App.tsx:6]: ✅

**Pull request merged**